src/main/java/co/cask/cdc/plugins/common/SparkUtil.java

```diff
@@ -0,0 +1,32 @@
+package co.cask.cdc.plugins.common;
+
+import org.apache.spark.api.java.JavaRDD;
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+/**
+ * Utility methods for dealing with Spark configuration and data.
+ */
+public class SparkUtil {
```

Comment:
nit: Lets call this SparkConfigs. We usually name utility classes as some plural category that describes what's in the class. This is to try and prevent ever growing utility classes where everything in it is unrelated.